sdk/daml-assistant/daml-helper/src/DA/Daml/Helper/Util.hs

```diff
@@ -334,7 +334,7 @@ cantonConfig CantonOptions{..} =
                      , "admin-api" Aeson..= port cantonDomainAdminApi
                      , "init" Aeson..= Aeson.object
                            [ "domain-parameters" Aeson..= Aeson.object
-                               [ "protocol-version" Aeson..= ("5" :: T.Text) ]
+                               [ "protocol-version" Aeson..= ("6" :: T.Text) ]
```

Comment:
have you tried 
```suggestion
                               [ "protocol-version" Aeson..= ("6" :: T.Text) ]
```
```suggestion
                               [ "protocol-version" Aeson..= ("latest" :: T.Text) ]
```

Reply:
This could randomly break though, as the acceptable versions in the canton file aren't automatic

Reply:
But i guess - we'd notice in the codedrop - sure